src/Components/Blazor/Build/test/DirectiveRazorIntegrationTest.cs

```diff
@@ -149,7 +149,7 @@ public class TestLayout : IComponent
             [Parameter]
             RenderFragment Body { get; set; }
 
-            public void Configure(RenderHandle renderHandle)
+            public void Attach(RenderHandle renderHandle)
```

Comment:
`Attach` seems better to me than `AttachToRenderer` because `Render` is already in the parameter list.